packages/react-jsx-highcharts/src/components/Hidden/Hidden.js

```diff
@@ -1,20 +1,7 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React from 'react';
 
-class Hidden extends Component {
-
-  static propTypes = {
-    children: PropTypes.node
-  };
-
-  render () {
-    const { children } = this.props;
-    if (!children) return null;
-
-    return [
-      children
-    ];
-  }
-}
+const Hidden = ({ children = null}) => (
```

Comment:
What is the purpose of <Hidden>? Could it be replaced with <React.Fragment>?

Reply:
It can yes, it was a legacy component from back before React.Fragment existed.